microcosm_flask/tests/conventions/test_alias.py

```diff
@@ -59,8 +46,8 @@ def setup(self):
         self.ns = Namespace(
             subject=Person,
         )
-        configure_crud(self.graph, self.ns, PERSON_MAPPINGS)
         configure_alias(self.graph, self.ns, PERSON_MAPPINGS)
+        configure_crud(self.graph, self.ns, PERSON_MAPPINGS)
```

Comment:
As noted by @ronanm-globality, docs elsewhere tell us we _must_ configure these in the "old" order, so the new changes seem a bit wrong - or at least we should update the docs.

(also unfortunate: these things seem to just quietly not work when done in the wrong order - which could make the migration frustrating for users